src/actors/upgrade/common/list_all_files/check

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from subprocess import Popen, PIPE
+import json
+import sys
+
+
+def log_error(errmsg):
+    """Log error on stderr."""
+    sys.stderr.write("%s\n" % errmsg)
+
+
+# NOTE: expected dirs without whitespaces
+cmd_local_mnt = "df --local -P | rev | cut -f1 -d' ' | rev | tail -n +2"
+cmd_file_list = "find $i -xdev -print"
+cmd = "for i in $(%s);do %s; done |sort |uniq" % (cmd_local_mnt, cmd_file_list)
+
+p = Popen(cmd, shell=True, stdout=PIPE)
+p_out, dummy = p.communicate()
+if p.returncode != 0:
+    log_error("Some unkown error during generating of file list.")
+    sys.exit(p.returncode)
+    # TODO: errror handling? it is highly unexpected to get error here
```

Comment:
even if you want to catch errors here, what will you do? From this what you wrote above, I think you can't do more, log error and exit, exactly what you did or raise Exception. What should be better? Still don't know, we didn't discuss how we should pass errors in this framework...

Reply:
I am not sure about the comment today as well :-) Maybe it was question whether I should generate such error message and provide `sys.exit` of `p.returncode` or completely remove this block of code and let it crash. Probably the comment can be removed.